WarframeObjects/Cetuscycle.cs

```diff
@@ -1,3 +1,5 @@
+using System;
+
```

Comment:
any particular reason for the extra import? doesn't look like it's always needed, like with Platform

Reply:
It's for System.DateTime. The project didn't compile for me